R/check.R

```diff
@@ -110,6 +111,14 @@ check_dataset <- function(d,suffix="",req=c("snp"),warn.minp=1e-6) {
   if(!(d$type %in% c("quant","cc")))
     stop("dataset ",suffix,": ","type must be quant or cc")
 
+  ## varbeta should be > 0
+  if("varbeta" %in% nd & !all(varbeta > 0))
+	  stop("varbeta should be strictly > 0")
+
+  ## MAF should be > 0
+  if("MAF" %in% nd & !all(MAF > 0))
+	  stop("MAF should be strictly > 0")
+
```

Comment:
Does this do something different to the code on lines 85-87?

Reply:
ha! good spot, no it doesn't
do I make a new commit and pull request now, or...?

Reply:
The new commit on the correct branch will auto-update the PR on github once you push it (the PR is "tracking" the branch it comes from)